shared/python/slack_util.py

```diff
@@ -14,14 +14,10 @@ def generate_ping_list(user_ids):
     return ', '.join(['<@{}>'.format(user_id) for user_id in user_ids])
 
 
-def build_field(title='', value='', tm_start=None, tm_end=None, duration=None, short=False):
+def build_field(title='', value='', short=False):
```

Comment:
@AD1024 I didn't pick up on this in the review of #45 but I think it's best if `slack_util` sticks to the data structures used for building Slack API messages; if we need to introduce a layer over it, I think we should keep it separate from these data structures

Reply:
Sure. Got it.